src/styles/personOversiktTheme.ts

```diff
@@ -4,33 +4,53 @@ const fadeIn = keyframes`
     from { opacity: 0 }
 `;
 
+const oneRemInPx = 16;
```

Comment:
Gjør ikke dette det nå umulig å dynamisk endre fontstørrelse? Ikke det at andre trenger det, men nå låser vi det?

Reply:
Nope, tror da ikke det. Intensjonen er hvertfall det motsatte ;-) Men for å beregne en rem-verdi må vi ha en "standardstørrelse" i px. Standardstørrelsen i designerne er 16px, dvs at 1rem = 16px. Poenget med denne funksjonen er å regne om det som brukes i designene (px) til rem nettop slik at det blir responsivt.